*** tests/A-08-01-M-ok.out	2019-10-28 12:43:38.000000000 +0000
--- tests/A-08-01-M-ok.outhyp	2019-11-18 10:47:31.455301300 +0000
***************
*** 14,28 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 0.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - aa - bb@cc.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,27 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 1 - aa - bb@cc.com - NORMAL - ATIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
***************
*** 30,45 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Novo utente criado com o número 1.
! Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 0 - aa - bb@cc.com - NORMAL - ACTIVO
! 1 - bb - cc@dd.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 29,43 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: Introduza o nome do utente: Introduza o endereço de correio do utente: Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
  3 - Listar utentes
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 1 - aa - bb@cc.com - NORMAL - ATIVO
! 2 - bb - cc@dd.com - NORMAL - ATIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
